.github/workflows/merge_gatekeeper.yml

```
@@ -16,3 +16,9 @@ jobs:
           score_rules: |
             #maintainer: 2
             #reviewer: 1
+      - uses: upsidr/merge-gatekeeper@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          self: merge_gatekeeper
+          # https://github.com/upsidr/merge-gatekeeper/issues/71#issuecomment-1660607977
+          ref: ${{ github.event.action == 'auto_merge_enabled' && github.event.pull_request.head.ref || github.ref }}
```

こんな感じにしたりすると、「PRのとき」という意図が伝わりやすいかも？
```suggestion
          ref: ${{ github.event.pull_request && github.event.pull_request.head.ref || github.ref }}
```

---

確かに。そうしてみます。